src/components/app/flyouts/settings/settings.js

```diff
@@ -152,7 +218,10 @@ export class Settings extends LinkedComponent {
                 </button>
               </Section.Content>
             </Section.Container>
-            <ApplicationSettings onUpload={this.onUpload} applicationNameLink={this.applicationName} {...this.props} />
+            <ApplicationSettings
```

Comment:
What changed here? If nothing is changed, there's no reason to add extra lines.

Reply:
Coding Standard: Max 90 to 110 chars per line